app/presenters/nsm/check_answers/cost_summary_card.rb

```diff
@@ -206,6 +207,14 @@ def name_row(name, work_type_changed)
           numeric: false
         }
       end
+
+      def allowed_total_cost(allowed_rows)
+        allowed_rows.sum(zero) { _1.allowed_total_cost || zero }
+      end
+
+      def zero
+        BigDecimal(0)
+      end
     end
```

Comment:
please could you add a quick comment re: why we are using the BigDecimal zero. I understand from your comments in the PRs channel but I feel like this might be one of those things that's hard to understand the intention of if you have no context

Reply:
Sure! Done.